app/components/works/dates_component.html.erb

```diff
@@ -72,7 +72,7 @@
                          date_clear_target: "approximate",
                          action: "date-approximate#change"},
                   class: "form-check-input" %>
-            <%= form.label "created(approx0)", "Approximate (can't be selected if 'day' is entered)" %>
+            <%= form.label "created_approx0_", "Approximate (can't be selected if 'day' is entered)" %>
```

Comment:
are these underscore changes needed in these form labels?

Reply:
Yes, it what makes them match. (Don't ask me to explain why ...)